javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,9 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  for (var i = 0; i < words.length; i++) {
+    var temp = words[i];
+    temp = temp + suffix;
+    words[i] = temp;
```

Comment:
You don't want to change the original array values, since that change will persist after the function ends. Instead, you want to make your own empty array and push the new values to that array, and return the new array at the end of the function.

Reply:
Hey Cody Thank you for the feedback I believe I fixed the issue now.